docs/guide/file_management.md

```diff
@@ -37,7 +37,7 @@ ez_io:
                 adapter: default
 ```
 
-The 'default' Flysystem adapter's directory is based on your site settings, and will automatically be set to `%ezpublish_legacy.root_dir%/$var_dir$/$storage_dir$` (for example: `/path/to/ezpublish_legacy/var/ezdemo_site/storage`).
+The 'default' Flysystem adapter's directory is based on your site settings, and will automatically be set to `%webroot_dir%/$var_dir$/$storage_dir$` (for example: `/path/to/ezplatform/web/var/site/storage`).
```

Comment:
The original path will still be automatically correct if legacy bridge is installed, afaik.

Reply:
> The original path will still be automatically correct if legacy bridge is installed, afaik.

Hmm, ok. I can make a note here. Because w/o LB this piece of the doc is incorrect.

Reply:
> > The original path will still be automatically correct if legacy bridge is installed, afaik.
> 
> Hmm, ok. I can make a note here. Because w/o LB this piece of the doc is incorrect.

Added in relevant places via 2455169. Dropped /var/site/storage (or /var/ezdemo_site/storage) suffix because the doc root in this case needs to be set to the web root so the given examples make sense.